app/models/record.rb

```diff
@@ -0,0 +1,12 @@
+class Record < ApplicationRecord
+  belongs_to :user
+  has_many :category_records
+  has_many :categories, through: :category_records
+
+  validates :name, presence: true, length: { maximum: 250 }
+  validates :amount, presence: true, numericality: { greater_than: 0 }
+
```

Comment:
- [ ] I was able to record without a category. Kindly note that all inputs are required, and a user must select AT LEAST ONE category when creating a transaction. Kindly add validations for the record category.

  ![tchilo-cate](https://user-images.githubusercontent.com/36542986/164297312-6564eb0f-8488-4900-a5ba-d9d5ff78f38e.png)